WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardViewModel.swift

```diff
@@ -160,6 +162,7 @@ final class BlazeCampaignDashboardViewModel: ObservableObject {
     }
 
     func didSelectCreateCampaign(source: BlazeSource) {
+        analytics.track(event: .DynamicDashboard.dashboardCardInteracted(type: .blaze))
```

Comment:
❓ Same question as above about the method not being triggered.

Reply:
<img src="https://github.com/woocommerce/woocommerce-ios/assets/524475/49027b09-2213-4c6d-af4f-3f40627fdbc2" width="320"/>

This event is triggered upon tapping on the Blaze campaign in the dashboard card.

Reply:
From my testing, tapping Create campaign triggers only `createBlazeCampaignTapped` in `DashboardViewHostingController`. I'm confused why we have two places to track the interaction for this button. Can you check again @selanthiraiyan?